### DISCUSSION TITLE
Pipewire 1.0.0 released!

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#363] [General]
"This release will soon arrive in stable Ubuntu, Fedora and Arch repositories, and you will receive the update as per the usual distribution channel."
https://debugpointnews.com/pipewire-1-0/
https://gitlab.freedesktop.org/pipewire/pipewire/-/releases/1.0.0

Just curious on how SpiralLinux will welcome and implement this novelty?

**Comment by geckolinux** [comment]
Hi there, PipeWire is already implemented and preconfigured specifically for low-latency tasks in SpiralLinux starting with version [12.231001](https://github.com/SpiralLinux/SpiralLinux-project/releases/tag/12.231001). Users that installed SpiralLinux from an ISO older than that might find it easier to re-install with the latest ISO if they need low-latency JACK audio compatibility, but the vast majority will probably be fine with the older PulseAudio configuration. Regarding the latest 1.0 release of PipeWire, it won't appear in Debian Stable for years until Debian 13 is released, but it will probably appear in Debian Stable Backports at some point for users that opt to switch to that version.

**Reply by dacomboman** [reply]
SpiralLinux sound is good as is.
Not being in the music industry, don't think i could notice the difference those techies do.
Find it encouraging to see how much work has gone into this Pipewire in so little time!
Spotted on MX forum that 1.0.0 is presently back-ported to their "test repo."

**Reply by geckolinux** [reply]
There's really not a lot of change between the version of PipeWire in Debian Stable and the latest 1.0 release. The lead developer basically said that it was about time to move to a stable version number as most of the showstopper issues have been resolved. But if you don't have a specific need or a specific issue with PulseAudio then PipeWire isn't even necessary, and if you do need a specific PipeWire feature then the version in Debian 12 when properly configured works great.
